doc/rbd/rbd-exclusive-locks.rst

```diff
@@ -6,19 +6,24 @@
 
 .. index:: Ceph Block Device; RBD exclusive locks; exclusive-lock
 
-Exclusive locks are a mechanism designed to prevent multiple processes
-from accessing the same Rados Block Device (RBD) in an uncoordinated
-fashion. Exclusive locks are heavily used in virtualization (where
-they prevent VMs from clobbering each others' writes), and also in RBD
-mirroring (where they are a prerequisite for journaling).
+Exclusive locks are mechanisms designed to prevent multiple processes from
+accessing the same Rados Block Device (RBD) in an uncoordinated fashion.
+Exclusive locks are used heavily in virtualization (where they prevent VMs from
+clobbering each other's writes) and in RBD mirroring (where they are a
+prerequisite for journaling).
 
-Exclusive locks are enabled on newly created images by default, unless
-overridden via the ``rbd_default_features`` configuration option or
-the ``--image-feature`` flag for ``rbd create``.
+By default, exclusive locks are enabled on newly created images. This default
+can be overridden via the ``rbd_default_features`` configuration option or the
+``--image-feature`` option for ``rbd create``.
 
-In order to ensure proper exclusive locking operations, any client
-using an RBD image whose ``exclusive-lock`` feature is enabled should
-be using a CephX identity whose capabilities include ``profile rbd``.
+.. note::
+   The RBD features ``object-map`` and ``fast-diff`` depend upon the
```

Comment:
`object-map` and `fast-diff` are just examples -- it's not an exhaustive list.  I'd say: "Many image features, for example ``object-map`` and ``fast-diff``, depend upon exclusive locking.  Disabling ``exclusive-lock`` feature will negatively affect the performance of some operations.

Reply:
@idryomov I've made this change, almost verbatim. I've left the change in a separate commit (https://github.com/ceph/ceph/pull/49488/commits/25dda5922ccb2d04b61830e5b36e3eff8db1760c), which I'll squash after I win your approval of the changes in this PR.

Reply:
Thanks, LGTM!